arm-software/embedded/samples/Makefile.conf

```diff
@@ -10,7 +10,7 @@ BIN_PATH=../../../bin
 endif
 
 MICROBIT_TARGET=--target=armv6m-none-eabi -march=armv6m -mfpu=none -mfloat-abi=soft
-AARCH64_TARGET=--target=aarch64-none-elf
+RASPI3B_TARGET=--target=aarch64-none-elf -mno-unaligned-access
```

Comment:
I would prefer not to lump together `-mno-unaligned-access` here with the target details, and have it added separately on the Makefile for each sample.

The reason that `-mno-unaligned-access` is needed is that unaligned access hasn't been enabled in the boot code, looks like it is executing from device memory. This is a property of the bootcode and not the target.

Given that the picolibc and llvm-libc sample use different bootcode we could vary `-mno-unaligned-access` independently.